bridge/qb/shared/main.lua

```diff
@@ -85,13 +87,15 @@ qbShared.CommaValue = lib.math.groupdigits
 qbShared.RandomStr = function(length)
     if length <= 0 then return '' end
     local pattern = math.random(2) == 1 and 'a' or 'A'
-    return qbShared.RandomStr(length - 1) .. lib.string.random(pattern)
+
+    return lib.string.random(pattern, length)
```

Comment:
Careful! The docs on [`lib.string.random`](https://overextended.dev/ox_lib/Modules/String/Shared#stringrandom) state that the `length` argument is only for padding it or omitting characters! It does not affect the pattern in any other way.
That is not the intention of `RandomStr`, so you should revert to the previous solution or create the pattern with a loop.